static/app/views/eventsV2/queryList.tsx

```diff
@@ -208,6 +225,21 @@ class QueryList extends React.Component<Props> {
           )}
           renderContextMenu={() => (
             <ContextMenu>
+              <Feature
+                organization={organization}
+                features={['connect-discover-and-dashboards']}
```

Comment:
We should check for both `dashboard-edit` and `connect-discover-and-dashboards` features here

Reply:
Ahh nice catch, forgot to include some of the feedback changes from the other PR into this one! Updated!